node/benchmarks/index.js

```diff
@@ -20,21 +20,28 @@
 
 'use strict';
 
+/*eslint no-console: 0*/
```

Comment:
In preparation of upgrading tchannel to uber-standard I'm writing all new code to be uber-standard compliant. Mostly relying on my text editor plugin.

I can remove the comment though.